nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/message/BulkStatus.java

```diff
@@ -312,6 +383,20 @@ public void setUsername(String username) {
         this.username = username;
     }
 
+    /**
+     * Gets the accumulated processing time in milliseconds.
+     */
+    public long getProcessingMillis() {
```

Comment:
Maybe find a more specific name like `getProcessingAccumulatedMillis`? Also why not use a `Duration` instead of millis? The other methods seem to use `java.time` objects this would be more consistent from an API standpoint